microsite/sidebars.js

```diff
@@ -342,13 +354,6 @@ module.exports = {
       'tooling/package-metadata',
     ],
     Deployment: [
-      'deployment/index',
-      'deployment/scaling',
-      'deployment/docker',
-      'deployment/k8s',
-      'deployment/heroku',
-      'deployment/koyeb',
-      'deployment/flightcontrol',
       {
         type: 'category',
         label: 'Using Backstage Deploy (alpha)',
```

Comment:
I think we can also remove this if I remember correctly from the last SIG? But was not 100% sure... 💭